packages/reputation-miner/ReputationMinerClient.js

```diff
@@ -367,30 +377,44 @@ class ReputationMinerClient {
 
         // If we're here, we do have an opponent.
         // Has our opponent timed out?
+        // TODO: Remove these magic numbers
```

Comment:
Does the client not have a `constants.js` file?

Reply:
It does not. We could add one, ideally though we'd want these values being picked up off the contracts, which I think is what I had in mind when I wrote this comment.